### ISSUE TITLE
Improve effectiveness!

### ISSUE THREAD

**Issue by NeroBlackstone** [JuliaMusic/PianoHands.jl#2]
Currently, in test set (PIG dataset 136-150) it has about 93% accuracy.
But it still works not very well in real world recorded midi.
We need better network structure and pre-trained weights.
